### ISSUE TITLE
Add function to turn annotations into one hot encoding

### ISSUE THREAD

**Issue by sprestrelski** [UCSD-E4E/PyHa#138]
[Example here](https://github.com/UCSD-E4E/passive-acoustic-biodiversity/blob/d34ca73673b1e73f5a284a040b39cdc1e473fafd/TweetyNET/cosmos_retraining.ipynb)

Idea:

|  | a | file.wav_3 | file.wav_6 | file.wav_9 | file.wav_12 | file.wav_15 | file.wav_18 |
|-|-|-|-|-|-|-|-|
| file1.wav | 1 | 0 | 0 | 1 | 1 | 0 | 1 |
| file1.wav | 1 | 0 | 0 | 1 | 1 | 0 | 1 |
| file1.wav | 1 | 0 | 0 | 1 | 1 | 0 | 1 |
Function for making annotations into one-hot encoding
weakly labeling

If you have 3-second chunks, have the name + offset
columns are all possible annotations that the chunk could have
1s where all of the annotations are
each column is a label, 1 signifies

**Comment by JacobGlennAyers** [comment]
I imagine that this would be for ease of use with a library such as PyTorch? I.E. it is useful when trying to use PyHa outputs as training data? 

What would be the added value of having that implemented natively in PyHa compared to using the existing functionality in deep learning frameworks?
